CRM/Case/Form/AddToCaseAsRole.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * Class CRM_Case_Form_AddToCaseAsRole
+ */
+class CRM_Case_Form_AddToCaseAsRole extends CRM_Contact_Form_Task {
+
+  /**
+   * @inheritdoc
+   */
+  public function buildQuickForm() {
+
+    $this->add('text', 'assign_to', ts('Assign to'));
```

Comment:
Your recent changes look good, but I'm still curious to know why the javascript is necessary instead of simply doing 

    $this->addEntityRef('assign_to', ts('Assign to'), array('create' => TRUE, 'api' => array('params' => array('contact_type' => 'Individual'))), TRUE);

Reply:
Looks good. But it appears you still need to remove this redundant form element.

Reply:
@colemanw,

I removed the redundant element, thanks for pointing it out. I also added a PR on user guide repo documenting the functionality.